### ISSUE TITLE
Issue with generics inheritance

### ISSUE THREAD

**Issue by valdisiljuconoks** [jburzynski/TypeGen#185]
There seems to be an issue with generating types having generic inheritance.

## Versions

TypeGen: 5.0.0
.NET 8

## Sample data model

```csharp
[ExportTsInterface]
public class CustomerDto
{
    public DateTime CreatedAt { get; set; }
}

[ExportTsInterface]
public class GetCustomersResponseDto : GenericServiceResponseDto<CustomerDto?>
{
}

[ExportTsInterface]
public abstract class GenericServiceResponseDto<T>
{
    [TsOptional]
    public T? Data { get; set; }
}
```

Generated `GetCustomersResponseDto` model:

```typescript
/**
 * This is a TypeGen auto-generated file.
 * Any changes made to this file can be lost when this file is regenerated.
 */

import { GenericServiceResponseDto } from './GenericServiceResponseDto.generated';
import { CustomerDto } from './CustomerDto.generated';

export interface GetCustomersResponseDto {

}
```

## Expected results
`GetCustomersResponseDto` to be generated extending `GenericServiceResponseDto`.

## Reproduction project
https://github.com/valdisiljuconoks/TypeGenInheritanceWithGenerics

**Comment by jburzynski** [comment]
This should be fixed in 5.0.1.

**Comment by valdisiljuconoks** [comment]
Can confirm that it's working as expected in 5.0.1. Thanks!
